Updated Amplify Hosting with Next.js docs - dynamic routes (#2794)
aws-amplify/docs: docs/guides/hosting/fragments/nextjs.md

$ git push origin master
```

## Dynamic routes

Next.js also supports dynamic routes.

Let's say you have a folder and file structure that looks like this:

```
/pages/posts/[id].js
```

> This component needs to read the ID of the URL parameter and use this data in some way in the app. To make this happen, we can use `next/router`:

Reviewer: ```suggestion
This component needs to read the ID from the URL and do something useful with it in the app. To make this happen, we can use `next/router`:
```

Author: Thanks for the feedback, just updated.
